hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/RemoteWithExtrasException.java

```diff
@@ -58,15 +60,26 @@
   }
 
   public RemoteWithExtrasException(String className, String msg, final boolean doNotRetry) {
-    this(className, msg, null, -1, doNotRetry);
+    this(className, msg, doNotRetry, false);
+  }
+
+  public RemoteWithExtrasException(String className, String msg, final boolean doNotRetry,
```

Comment:
Oh, it pains me so that this class is `IA.Public` and it extends a Hadoop class :'(

Reply:
Yea I think it'd be cool to remove/deprecate this class if we were to refactor exception handling based on some of the discussion above.